src/rpcnames.cpp

```diff
@@ -435,6 +440,106 @@ name_filter (const UniValue& params, bool fHelp)
 /* ************************************************************************** */
 
 UniValue
+name_pending (const UniValue& params, bool fHelp)
+{
+  if (fHelp || params.size () > 1)
+    throw std::runtime_error (
+        "name_pending (\"name\")\n"
+        "\nList unconfirmed name operations in the mempool.\n"
+        "\nIf a name is given, only check for operations on this name.\n"
+        "\nArguments:\n"
+        "1. \"name\"        (string, optional) only look for this name\n"
+        "\nResult:\n"
+        "[\n"
+        "  {\n"
+        "    \"op\": xxxx       (string) the operation being performed\n"
+        "    \"name\": xxxx     (string) the name operated on\n"
+        "    \"value\": xxxx    (string) the name's new value\n"
+        "    \"txid\": xxxx     (string) the txid corresponding to the operation\n"
+        "    \"ismine\": xxxx   (boolean) whether the name is owned by the wallet\n"
+        "  },\n"
+        "  ...\n"
+        "]\n"
+        + HelpExampleCli ("name_pending", "")
+        + HelpExampleCli ("name_pending", "\"d/wikileaks\"")
```

Comment:
Minor nitpick: d/wikileaks is, to our knowledge, not controlled by WikiLeaks, therefore we probably shouldn't use it as an example.